compiler/internal/analysis/info_test.go

```diff
@@ -1529,21 +1694,29 @@ func (bt *blockingTest) assertFuncInstCount(expCount int) {
 	if got := bt.pkgInfo.funcInstInfos.Len(); got != expCount {
```

Comment:
Here and other assert* functions should call t.Helper(), so that the test error shows the line in the surrounding test case and not the helper itself.

Reply:
Good point, I'll add that in as part of #1350

Reply:
Added https://github.com/gopherjs/gopherjs/pull/1350/commits/1fa608f5b1f374773e22d94c5d61d349b78137bd